### DISCUSSION TITLE
Create a new line in slide title when rendering to powerpoint

### DISCUSSION THREAD

**Discussion by DanOvando** [quarto-dev/quarto-cli#3647] [Q&A] [answered]
For reasons that don't warrant getting into, I need to be able to break the titles of my slides up across two lines. I can get it to work for revealjs and beamer, but not when rendering to powerpoint. 

When rendering to `beamer`, this works just fine

```
## Stuff on Top Line \newline Stuff On Bottom Line

Slide Content

```

And when rendering using `revealjs` this works just fine

```
## Stuff on Top Line <br> Stuff On Bottom Line

Slide Content

```

But, neither work when rendering to pptx. I know that pptx is a finicky beast, but any thoughts on whether this is possible, or better yet a output-neutral way of doing this?

**Comment by baptiste** [comment, marked as answer]
You can use @tarleb 's Lua filter to convert `<br>` into a hard break regardless of the output format: https://stackoverflow.com/a/52193443

```
--- Transform a raw HTML element which contains only a `<br>`
-- into a format-indepentent line break.
function RawInline (el)
  if el.format:match '^html' and el.text:match '%<br ?/?%>' then
    return pandoc.LineBreak()
  end
end
```

**Reply by DanOvando** [reply]
That does it, thanks! For others that come across this, the trick then if you're not rendering from command line is to save the above code as a `.lua` file, and then include it as a [filter](https://quarto.org/docs/extensions/filters.html) in the YAML of your quarto document. 

```
---
title: "Something You Need Line Breaks"
format:
  pptx: default
filters:
  - linebreaks.lua
---

##  Stuff on Top Line <br> Stuff On Bottom Line

Slide Content


```
